speechly-ios-client.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -15,8 +15,13 @@
         "repositoryURL": "https://github.com/grpc/grpc-swift.git",
         "state": {
           "branch": null,
+<<<<<<< HEAD
```

Comment:
Merge not done here, as the build status shows also. Maybe just grab the latest from master?

Reply:
Yeah, will do!